### ISSUE TITLE
A tutor should receive a notification in the dashboard when they get a new request

### ISSUE THREAD

**Issue by jlvivero** [TutorFox/TutorFox#32]

**Comment by A-deLuna** [comment]
I'm thinking for this we should add a property to the state in line 30 that stores the number of seen tutoringRequests, then make a function that every 5 seconds makes a request and checks how many tutoringRequests there are.

```
if   seen < total: 
     add show a small icon with the number of pending requests
     (total - seen ) to the button that switches tabs. Also add a function
     that triggers when that button is clicked and updates
     the state seen = total 
```

also make a function in Dashboard that substracts one from seen, and pass that function down to RequestPanel so that its triggered when a user clicks the close button

**Comment by A-deLuna** [comment]
fixed in #53
